src/components/ImportSpreadsheet.tsx

```diff
@@ -94,7 +94,9 @@ function ImportSpreedsheet({backTo, goTo}: ImportSpreedsheetProps) {
                 const workbook = XLSX.read(new Uint8Array(arrayBuffer), {type: 'buffer'});
                 const worksheet = workbook.Sheets[workbook.SheetNames[0]];
                 const data = XLSX.utils.sheet_to_json(worksheet, {header: 1, blankrows: false});
-                setSpreadsheetData(data as string[][])
+                const formattedSpreadsheetData = data.map((row) => (Array.isArray(row) ? row.map((cell) => (typeof cell === 'string' ? cell : String(cell))) : []));
```

Comment:
```suggestion
                const formattedSpreadsheetData = data.map((row) => row.map((cell) => String(cell));
```

is there a possibility where the row isn't an array?

Reply:
@getusha Because data is unknown[], I added Array.isArray(row) to fix the ESLint error.

Reply:
yes